bin/test.py

```diff
@@ -0,0 +1,109 @@
+import os
+import subprocess
+
+import pytest
+import testinfra
+import yaml
+
+ASTRO_IMAGE_NAME = os.environ["ASTRO_IMAGE_NAME"]
+ASTRO_IMAGE_TEST_CONFIG_PATH = os.environ["ASTRO_IMAGE_TEST_CONFIG_PATH"]
+
+os.environ["ASTRO_IMAGE_TAG"] = os.environ.get("CIRCLE_SHA1", "latest")
+test_config = {}
+
+# Read the test config
+if os.path.exists(ASTRO_IMAGE_TEST_CONFIG_PATH):
+    with open(ASTRO_IMAGE_TEST_CONFIG_PATH) as file:
+        config = yaml.safe_load(file)
+
+        if config is not None:
+
+            # Reading test config
+            if "tests" in config:
+                test_config = config["tests"]
```

Comment:
Change suggested by sourcery:
```suggestion
        if config is not None and "tests" in config:
            test_config = config["tests"]
```

Reply:
Yes, we can as only one condition for now to test here.

Reply:
Done